src/resolve/resolved-varlink.c

```diff
@@ -37,6 +37,22 @@ static void lookup_parameters_destroy(LookupParameters *p) {
         free(p->name);
 }
 
+static int dns_query_new_for_varlink(
+                Manager *m,
+                DnsQuery **ret,
+                DnsQuestion *question_utf8,
+                DnsQuestion *question_idna,
+                DnsPacket *question_bypass,
+                sd_varlink *link,
+                int ifindex,
+                uint64_t flags) {
+
+        int q = dns_query_new(m, ret, question_utf8, question_idna, question_bypass, ifindex, flags);
```

Comment:
Please use `int r`, rather than `q`.
In our code base `int r` is a kind of special variable. Please use it.